assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -9412,7 +9412,7 @@ public void doDelete_assignment(RunData data) {
                 String title = assignment.getTitle();
 
                 // remove rubric association if there is one
-                rubricsService.deleteRubricAssociation(RubricsConstants.RBCS_TOOL_ASSIGNMENT, id);
+                rubricsService.softDeleteRubricAssociation(RubricsConstants.RBCS_TOOL_ASSIGNMENT, id);
```

Comment:
feel like we don't need to softly delete a rubric association at all?

Reply:
This is done to unlock the rubric and allow removing it, keeping consistency with the approach taken on https://github.com/sakaiproject/sakai/pull/7749